chart/templates/mayastor/obs/obs-callhome-deployment.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.obs.callhome.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-obs-callhome
+  labels:
+    app: obs-callhome
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: obs-callhome
+  template:
+    metadata:
+      labels:
+        app: obs-callhome
+    spec:
+      serviceAccount: {{ .Release.Name }}-service-account
+      imagePullSecrets:
+        {{- include "base_pull_secrets" . }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- toYaml .Values.nodeSelector | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: obs-callhome
+          image: "{{ .Values.image.registry }}/{{ .Values.image.repo }}/mayastor-obs-callhome:{{ .Values.image.tag }}"
+          args:
+            - "-e http://{{ .Release.Name }}-api-rest:8081"
+            - "-n {{ .Release.Namespace }}"
+          env:
+            - name: RUST_LOG
+              value: {{ .Values.obs.callhome.logLevel }}
+            {{- if default .Values.base.logSilenceLevel .Values.obs.callhome.logSilenceLevel }}
+            - name: RUST_LOG_SILENCE
```

Comment:
Unless I'm mistaken, I don't think call home processes this env variable

Reply:
Right!